src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -17,19 +17,19 @@ namespace Bicep.LanguageServer
 {
     public class BicepCompilationManager : ICompilationManager
     {
-        private readonly ILanguageServer server;
+        private readonly ILanguageServerFacade server;
```

Comment:
This took me forever to figure out. It was failing on startup with a useless exception. Ended trying a ton of different things and eventually read the omnisharp "breaking changes" PRs where they mentioned this change 🤦‍♂️.